terraform/modules/hub/modules/ecs_asg/files/cloud-init.sh

```diff
@@ -158,3 +158,24 @@ EOF
 systemctl daemon-reload
 systemctl enable prometheus-node-exporter
 systemctl restart prometheus-node-exporter
+
+cat <<EOF > /usr/bin/instance-reboot-required-metric.sh
+#!/usr/bin/env bash
+
+echo '# HELP node_reboot_required Node reboot is required for software updates.'
+echo '# TYPE node_reboot_required gauge'
+if [[ -f '/run/reboot-required' ]] ; then
+  echo 'node_reboot_required 1'
+else
+  echo 'node_reboot_required 0'
+fi
+EOF
+
+chmod +x /usr/bin/instance-reboot-required-metric.sh
+
+apt-get install --yes moreutils
+
+cat <<EOF | crontab -
```

Comment:
```suggestion
crontab - <<EOF
```
we don't need `cat` here 🚫 🐈

Reply:
But I like cats... 🐈